samples/create_training_pipeline_image_classification_sample.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START aiplatform_create_training_pipeline_image_classification_sample]
+from google.cloud import aiplatform
+from google.protobuf import json_format
+from google.protobuf.struct_pb2 import Value
+
+
+def create_training_pipeline_image_classification_sample(
+    project: str,
+    display_name: str,
+    dataset_id: str,
+    model_display_name: str,
+    location: str = "us-central1",
+):
+    client_options = {"api_endpoint": "us-central1-aiplatform.googleapis.com"}
+    # Initialize client that will be used to create and send requests.
+    # This client only needs to be created once, and can be reused for multiple requests.
+    client = aiplatform.gapic.PipelineServiceClient(client_options=client_options)
+    training_task_inputs_dict = {
+        "multiLabel": True,
+        "modelType": "CLOUD",
+        "budgetMilliNodeHours": 8000,
```

Comment:
to my untrained eye this is a big scary number - will it cost the customer lots of money? Is there a reason for choosing this number in particular?

Reply:
TODO: Add link to docs for node hours